clients/csharp-uwp/UWPVoiceAssistantSample/DirectLineSpeechDialogBackend.cs

```diff
@@ -108,75 +117,89 @@ public Task InitializeAsync(StorageFile keywordFile)
         {
             Contract.Requires(keywordFile != null);
 
-            // Default values -- these can be updated
-            this.ConnectorConfiguration = this.CreateConfiguration();
-            this.ConfirmationModel = KeywordRecognitionModel.FromFile(keywordFile.Path);
+            var configRefreshRequired = this.RefreshConfigValues();
 
-            this.connectorInputStream = AudioInputStream.CreatePushStream();
-            this.connector = new DialogServiceConnector(
-                this.ConnectorConfiguration,
-                AudioConfig.FromStreamInput(this.connectorInputStream));
+            var refreshConnector = configRefreshRequired || (this.keywordFilePath != keywordFile.Path);

```

Comment:
Do you think we should just treat this as another config and pass around a bool instead? That'd simplify the object model a little, too (no weird "ConfirmationModel is an object" thing).

Doesn't need to be now, though.

Reply:
Not sure I follow this one? "Treat *this* as another config"- what is "this"?